requirements/tox.in

```diff
@@ -0,0 +1,3 @@
+# Requirements for using tox.
```

Comment:
We usually create a `travis.in` file instead, that also includes codecov and tox-battery.